### PR TITLE
Re added russian language

### PR DESCRIPTION
**Pull request by rovecode**

### CODE REVIEW

**Comment by yaichenbaum** [comment]
This adds back some strings that have been removed in previous commits, can you pull in the latest from the `develop` branch?

**Comment by rovecode** [comment]
> This adds back some strings that have been removed in previous commits, can you pull in the latest from the `develop` branch?

ok

**Comment by yaichenbaum** [comment]
Thank you so much!

File: Files/MultilingualResources/Files.ru-RU.xlf [hunk 1]
```diff
@@ -190,6 +242,130 @@
...
+          <target state="new">Имя</target>
+        </trans-unit>
+        <trans-unit id="RibbonAreaSortBySize.Text" translate="yes" xml:space="preserve">
+          <source>Size</source>
+          <target state="new">Размер</target>
+        </trans-unit>
+        <trans-unit id="RibbonAreaSortByType.Text" translate="yes" xml:space="preserve">
+          <source>Type</source>
+          <target state="new">Тип</target>
+        </trans-unit>
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ru-RU.xlf]: In the latest changes of `develop` these strings have been removed.

File: Files/MultilingualResources/Files.ru-RU.xlf [hunk 2]
```diff
@@ -242,166 +418,6 @@
           <source>Videos</source>
           <target state="new">Видео</target>
         </trans-unit>
-        <trans-unit id="ExtractFilesDialog.Title" translate="yes" xml:space="preserve">
```
Comment (yaichenbaum) [inline Files/MultilingualResources/Files.ru-RU.xlf]: These strings should not be removed. If you pull the latest from `develop` it should fix the issue.

Comment (rovecode) [reply Files/MultilingualResources/Files.ru-RU.xlf]: I fix all. I need some type to test application

**Comment by yaichenbaum** [comment]
I will test this locally to make sure it's working correctly.

**Comment by yaichenbaum** [comment]
@rovecode Thank you!

**Pull request merged**